CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 ### Changed
 
 - UI: Accessibility - Add ARIA role `button` to "Resend link" and "Cancel" links on Cross Device flow SMS Sent, Mobile Connected screens.
+- Internal: remove handles for `sdk_warnings` in responses.
```

Comment:
Minor: Maybe should add a note also that this property has been deprecated in the API responses, that's why it's being removed.